app/src/Homepage/Homepage.tsx

```diff
@@ -152,11 +152,58 @@ export const HomePage: React.FC = () => {
                 seen the material in the industry?
               </RatingDesc>
             </div>
+            <ClassCard name='Computer Communication and Networks' num={561} desc='Fundamentals of data transmission: coding, message formats, and protocols. Organization of computer networks. Examples of network implementations. Prerequisite: either CSE 451 or equivalent.' rating1={2.5} rating2={3} rating3={5}/>
```

Comment:
Is there a reason why we're hardcoding this in?

Reply:
OOPs haha no that was just a test card that I put on the homepage for easy CSS viewing -- forgot to remove